### PR TITLE
Add DB support for controlplane data in dataplane side

### PR DESCRIPTION
**Pull request by AmaliMatharaarachchi**
## Purpose
Fixes https://github.com/wso2/apk/issues/2019

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: common-controller/cmd/main.go [hunk 1]
```diff
@@ -30,6 +31,11 @@ func main() {
 	logger.Info("Starting the Web server")
 	go web.StartWebServer()
 	go server.StartInternalServer()
+	if conf.CommonController.Database.Enabled {
```
Comment (Krishanx92) [inline common-controller/cmd/main.go:34]: Can this be nil? CommonController.Database

Comment (AmaliMatharaarachchi) [reply common-controller/cmd/main.go:34]: No, it will be handled by default configs

File: common-controller/internal/operator/operator.go [hunk 2]
```diff
@@ -164,14 +170,17 @@ func InitOperator() {
...
 	}
-	config := config.ReadConfigs()
+
 	if config.CommonController.ControlPlane.Enabled {
 		go func() {
 			var controlPlane controlplane.ArtifactDeployer
 			if config.CommonController.ControlPlane.Persistence.Type == "K8s" {
 				controlPlane = controlplane.NewK8sArtifactDeployer(mgr)
 
+			} else if config.CommonController.ControlPlane.Persistence.Type == "DB" {
```
Comment (CrowleyRajapakse) [inline common-controller/internal/operator/operator.go:180]: Need to add configs to helm charts values.yaml

Comment (AmaliMatharaarachchi) [reply common-controller/internal/operator/operator.go:191]: fixed

**Review by Krishanx92 — APPROVED** [review]

**Pull request merged**